CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 ## [x.x.x]
 
 ### Added
+- A new blog post for version 4.17
```

Comment:
I'm not so sure this is needed, but don't mind much either way. The actual code was not touched, but it's never wrong to document.